coderd/database/dbmem/dbmem.go

```diff
@@ -7921,6 +7950,14 @@ func (*FakeQuerier) UpsertTailnetTunnel(_ context.Context, arg database.UpsertTa
 	return database.TailnetTunnel{}, ErrUnimplemented
 }
 
+func (q *FakeQuerier) UpsertTemplateUsageStats(_ context.Context) error {
+	q.mutex.Lock()
+	defer q.mutex.Unlock()
+
+	// TODO(mafredri): Implement.
+	panic("not implemented")
```

Comment:
I guess  I can see the implementation later in the stack?

Reply:
Nope! It'll appear here (this is still a draft after all) 😄